config/env.production

```diff
@@ -31,6 +31,9 @@ PORTAINER_HOST=portainer.telescope.cdot.systems
 # The host where supabase studio runs (e.g., https://supabase.telescope.cdot.systems)
 SUPABASE_HOST=supabase.telescope.cdot.systems
 
+# The Supabase API URL
+SUPABASE_URL=https://api.telescope.cdot.systems/v1/supabase
```

Comment:
This gives `404 page not found`

Reply:
Is there some step we have to do in order to enable Supabase on production that we already did on staging?  cc @Kevan-Y

Reply:
https://supabase.telescope.cdot.systems/ supabase studio is already enabled.

Reply:
I don't think we need anything, it should be enables automatically via Treafik

Reply:
https://api.telescope.cdot.systems/v1/supabase is working now, after @manekenpix's fix on prod, can you re-review?